### ISSUE TITLE
Compile error causes large branch of tests to be silently ignored

### ISSUE THREAD

**Issue by horyd** [Streamotion/vizard#8]
Reproducible on Ionic `develop`

With no issues:
`Found 766 tests to run`

Add a random line of nonsense code to the Atom Button 1's vizard file:
`Found 499 tests to run`

A rough look at that number suggests that it is excluding all `atoms` when it can't compile a single atom's vizard file

**Comment by horyd** [comment]
Fixed in horyd:bugfix/show-page-errors

Actually not a compile error, browserify doesn't see it, it's only when the script is run inside the puppeteer window that an error within the specific puppeteer page. Bubbling those errors onto the main vizard process is the key.

I've noticed that this will prevent any runtime errors in bootstrapping the tests, but won't stop any errors that are encountered during the running of the individual suites/tests. It will however still log them out for the user to see.
